src/hash.cr

```diff
@@ -13,8 +13,8 @@ class Hash(K, V)
 
   def initialize(block : (Hash(K, V), K -> V)? = nil, initial_capacity = nil)
     initial_capacity ||= 11
-    initial_capacity = 11 if initial_capacity < 11
     initial_capacity = initial_capacity.to_i
+    raise ArgumentError.new("Hash capacity must be positive") if initial_capacity <= 0
```

Comment:
I don't think it should raise if `initial_capacity == 0`

The user might want to create a Hash with initial capacity of 0. Even if it seems pointless, there might be some cases where it is usefull!